pricemap-typescript/Dockerfile

```diff
@@ -14,9 +14,9 @@ RUN if ! $(awk -F':' '{print $3}' /etc/group |grep -q ${GROUP_ID}) ; then groupa
 RUN usermod -a -G ${GROUP_ID} pricemap
 
 USER pricemap
-WORKDIR /home/pricemap/app
 
-COPY package*.json ./
-RUN npm ci
+COPY --chown=pricemap:${GROUP_ID} package*.json /home/pricemap/app/
+COPY --chown=pricemap:${GROUP_ID} . /home/pricemap/app
```

Comment:
Mmh this is not ideal as you would run `npm ci` if any file changes and you try to rebuild the container, while it may not be necessary.

Why did you have to run the deps installation after copying the source code?